src/components/profiles-modals/ImportProfileModal.vue

```diff
@@ -34,6 +34,7 @@ export default class ImportProfileModal extends mixins(ProfilesMixin) {
     private profileImportFilePath: string | null = null;
     private profileImportContent: ExportFormat | null = null;
     private profileMods: {known: ThunderstoreCombo[], unknown: string[]} = {known: [], unknown: []};
+    private partialImportCheckbox: boolean = false;
```

Comment:
Minor: `allowPartialImport` or `isPartialImportAllowed` would IMO be a better name